### PR TITLE
CMR-6931: Moved schema changes to v1.1 instead of altering 1.0 directly

### PR DESCRIPTION
**Pull request by mitchstartzel**
* In my 6931 PR, the UMM-Sub v1.0 PR was edited directly. This was a mistake. Instead, should have created a v1.1 - this ticket does just that.

### CODE REVIEW

**Review by ygliuvt — CHANGES_REQUESTED** [review]
The latest supported subscription version needs to be updated to `1.1` in `cmr.ingest.config`.

File: umm-spec-lib/resources/json-schemas/subscription/umm/v1.0/umm-sub-json-schema.json [hunk 1]
```diff
@@ -26,18 +26,20 @@
       "description": "The collection concept id of the granules subscribed.",
       "type": "string",
       "minLength": 1,
-      "maxLength": 255
+      "maxLength": 255
```
Comment (jaybarra) [inline umm-spec-lib/resources/json-schemas/subscription/umm/v1.0/umm-sub-json-schema.json:29]: Check your editors whitespace cleanup setting. Looks like it's adding spaces

**Review by jaybarra — APPROVED** [review]

File: umm-spec-lib/resources/json-schemas/subscription/umm/v1.0/umm-sub-json-schema.json (hunk 1 above)
Comment (mitchstartzel) [reply umm-spec-lib/resources/json-schemas/subscription/umm/v1.0/umm-sub-json-schema.json:29]: I tried to figure this out before - will look more into it.

**Comment by sxu123** [comment]
Do we need migration between 1.0 and 1.1?

**Review by ygliuvt — APPROVED** [review]

**Comment by ygliuvt** [comment]
How are you handling the existing subscription record? Are we delete them all?

**Comment by mitchstartzel** [comment]
> How are you handling the existing subscription record? Are we delete them all?

Is the alternative to this, "build a proper migration?" If so, I'm thinking that is the preferred option - but we should discuss!

**Review by jceaser — CHANGES_REQUESTED** [review]
no documentation needed to be changed?

**Comment by mitchstartzel** [comment]
Going to rework this with a proper migration, and link it to the correct ticket number as well.

**Pull request closed without merging**